src/Symfony/Component/Encryption/README.md

```diff
@@ -0,0 +1,20 @@
+Encryption Component
+====================
+
+The Encryption Component is an opinionated, high level abstraction over encryption
+libraries and PHP's Sodium extension. Encryption should be understandable without
```

Comment:
`over encryption libraries and PHP's Sodium extension`
I see the sodium but not `encryption libraries`, maybe a leftover a new dev?